hi-ml-cpath/src/health_cpath/models/deepmil.py

```diff
@@ -41,7 +41,7 @@ def __init__(self,
                  class_names: Optional[Sequence[str]] = None,
                  dropout_rate: Optional[float] = None,
                  verbose: bool = False,
-                 ckpt_run_id: Optional[str] = None,
+                 ssl_ckpt_run_id: Optional[str] = None,
```

Comment:
Why is this necessarily SSL ckpt? Could it not also be one of the imagenet encoders?

Reply:
Because the only weights we load from AML runs are SSL checkpoints, it's used here [encoder](https://github.com/microsoft/hi-ml/blob/d7c5ebf32f869cf56b898fc2150be6965f16f8c1/hi-ml-cpath/src/health_cpath/utils/deepmil_utils.py#L75) to create the SSLEncoder.
It can definitely be an imagenet encoder but we don't need an aml_run_id for these.